packages/sdkgen/sdkgen-demo-template/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
should we have copyright headers on template files? I lean towards no
The scripts, yes (hmm they don't have headers... because they are .js? because they aren't under src/?)